syft/format/common/spdxhelpers/to_format_model.go

```diff
@@ -742,14 +749,28 @@ func toOtherLicenses(catalog *pkg.Collection) []*spdx.OtherLicense {
 	slices.Sort(ids)
 	for _, id := range ids {
 		license := licenses[id]
+		value := license.Value
+		// handle cases where LicenseRef needs to be included in hasExtractedLicensingInfos
+		if license.Value == "" {
+			value, _ = strings.CutPrefix(license.ID, "LicenseRef-")
+		}
 		result = append(result, &spdx.OtherLicense{
 			LicenseIdentifier: license.ID,
-			ExtractedText:     license.Value,
+			ExtractedText:     value,
 		})
 	}
 	return result
 }
 
+// isSingularLicenseRef checks if the string is a singular LicenseRef-* identifier
+func isSingularLicenseRef(s string) bool {
```

Comment:
this seems to detect if there is a license ref, but how does it really help determine if it's singular or not? I think I'm missing some context here